test/runtests.jl

```diff
@@ -1,62 +1,64 @@
+Pkg.checkout("StaticArrays", "sd/fixedsizearrays")
```

Comment:
Ah true, since there are also 0.5 compatible StaticArray versions... I forgot about that, since GeometryTypes relies on all the new stuff in StaticArrays, which only comes on 0.6